spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'hawk'
+require 'byebug'
```

Comment:
To be Ruby <2.1 compatible we should also get rid of all the `polyfills/` directory that uses Refinements. They can be removed now as we are requiring `active_support`.